src/ec/packages/ec-component-tag/tag.story.js

```diff
@@ -1,63 +1,45 @@
-import merge from 'deepmerge';
+/* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
-
+import { getExtraKnobs, buttonLabels } from '@ecl-twig/story-utils';
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import dataLink from './demo/data--link';
 import dataButton from './demo/data--button';
 import dataRemovable from './demo/data--removable';
 import tag from './ecl-tag.html.twig';
 import notes from './README.md';
 
+// Preserve the adapted specs.
+const prepareTag = data => {
+  data.default_icon_path = defaultSprite;
+  data.tag.label = text('label', data.tag.label, buttonLabels.required);
+  if (data.tag.path) {
+    data.tag.path = text('Url', data.tag.path, buttonLabels.required);
+  }
+  if (data.tag.aria_label) {
+    data.tag.aria_label = text(
+      'Aria label',
+      data.tag.aria_label,
+      buttonLabels.required
+    );
+  }
+  getExtraKnobs(data);
+
+  return data;
```

Comment:
```
{#
  Parameters:
   - "tag" (associative array) (default: predefined structure): format:
      {
        type: 'link' (can be 'link', 'button', 'removable'),
        path: '' (tag url if needed),
        label: '' (tag text as string),
        aria_label: '' (optional) (aria label for removable tag as a string)
      }
    - "default_icon_path" (string) Default path for icon tag (need to render Icon component)
    - "extra_classes" (string) (default: '')
    - "extra_attributes" (array) (default: []): format: [
        {
          "name" (string) (default: ''),
          "value" (optional) (string)
        ...
      ]
#}
```

The idea would be that through the knobs you would be able to rebuild the entire input data, actually the see it interacting with it when it makes sense to do it, if you want to edit things for then copying the include code in the notes tab.
So here for the tag we would also have:
```
type as a select with the only option being the current variant
default_icon_path being again a select with the only option being the current value
```